spring-cloud-task-core/src/main/java/org/springframework/cloud/task/listener/TaskExecutionMeters.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2022-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.task.listener;
+
+import io.micrometer.common.docs.KeyName;
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.docs.DocumentedMeter;
+
+/**
+ * Enumeration for task execution metrics.
+ *
+ * @author Glenn Renfro
+ * @since 3.0.0
+ */
+public enum TaskExecutionMeters implements DocumentedMeter {
+	/**
+	 * Metrics created around a task execution.
+	 */
+	TASK_ACTIVE {
+		@Override
+		public String getName() {
+			return "spring.cloud.task.active";
+		}
+
+		@Override
+		public Meter.Type getType() {
+			return Meter.Type.LONG_TASK_TIMER;
+		}
+
+
+		@Override
+		public String getPrefix() {
+			return "spring.cloud.task";
+		}
+	};
+
+	public enum TaskTags implements KeyName {
```

Comment:
How about `TaskKeyNames`?
```suggestion
	public enum TaskKeyNames implements KeyName {
```